scripts/generate_root_ca_certs

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+# Commit that updated the certificates for Android 10.
+readonly COMMIT="be85a54767b3559e7bb262e4f1a78e1c5829e478"
+
+readonly DOWNLOAD_DIR="${SCRIPTS_DIR}/../target/tmp"
+readonly DOWNLOAD_FILE="${DOWNLOAD_DIR}/files.tar.gz"
+readonly EXTRACT_DIR="${DOWNLOAD_DIR}/files"
+readonly TARGET_FILE="${SCRIPTS_DIR}/../oak/server/rust/oak_loader/src/certs/roots.pem"
+
+if [[ -d "${DOWNLOAD_DIR}" ]]; then
+    rm --recursive --force "${DOWNLOAD_DIR}"
+fi
+
+if [[ -f "${TARGET_FILE}" ]]; then
+    rm "${TARGET_FILE}"
+fi
+
+curl \
+    --location "https://android.googlesource.com/platform/system/ca-certificates/+archive/${COMMIT}/files.tar.gz" \
+    --output "${DOWNLOAD_FILE}" \
+    --create-dirs
+
+mkdir "${EXTRACT_DIR}"
+
+tar --extract --gzip --file="${DOWNLOAD_FILE}" --directory="${EXTRACT_DIR}"
+
+files="$(find ./"${EXTRACT_DIR}"/*)"
+
+for file in ${files}; do
```

Comment:
Could you add some comments to explain what's going on here? e.g. that this is concatenating all the certs in a single pem file one by one, and perhaps point to some documentation that says that this is fine to do (in my ignorance, I would not have expected that I can just concatenate them together like this)

Reply:
Added comments.